### DISCUSSION TITLE
Technology of public sandbox likely to become basis of Swiss eID?

### DISCUSSION THREAD

**Discussion by ljakob** [e-id-admin/public-sandbox-trustinfrastructure#2] [Q&A] [answered]
Hi,

maybe I've not read all documentation: but I'd like to know how likely it is that Hyperledger Indy is going to become the technical platform for the Swiss eID?

Or is it more an experiment?

Thanks
Leif

**Comment by bit-sbonzelet** [comment, marked as answer]
Hello @ljakob

Indy/Anoncreds is our current sandbox stack and may not be the final stack or even the stack of future sandbox environments.  
We do evaluate different technologies to select the most suited for our requirements.  

The goal of the sandbox is to gather first hands on experience with operations, governance and administrative processes in the SSI context.

Kind Regards
 S. Bonzelet

**Comment by schischuMM** [comment]
**Hiermit kann jeder ein PoC erstellen...  ** __Open Source_ evtl. ein Blick wert..._

- https://github.com/Metis-DAO/Metis
- https://app.disco.xyz/ --- https://docs.disco.xyz/
- https://www.nuvosphere.io/ --- https://docs.nuvosphere.io/
- https://metis.io/ --- https://docs.metis.io/meta/ 
https://metisdao.medium.com/contribution-and-reputation-based-governance-structure-47c606ace4f6
https://metisdao.medium.com/introducing-hybrid-rollups-8da384225b5e

- https://github.com/0xPolygonID
- https://github.com/iotexproject !!!!

- https://www.kilt.io/
---https://github.com/KILTprotocol/sdk-js

- https://poap.xyz/how-to-use-poap
---https://discourse.poap.xyz/

- https://www.lens.xyz/
---https://docs.lens.xyz/docs

- https://github.com/walt-id 
- usw.
